ui/src/notebooks/shared/Resizer.tsx

```diff
@@ -142,18 +147,26 @@ const Resizer: FC<Props> = ({
 
   let body = children
 
-  if (!resizingEnabled) {
-    body = <div className="panel-resizer--empty">{emptyText}</div>
-  }
+  if (error) {
+    body = <div className="panel-resizer--error">{error}</div>
+  } else {
+    if (!resizingEnabled) {
+      body = <div className="panel-resizer--empty">{emptyText}</div>
+    }
 
-  if (resizingEnabled && visibility === 'hidden') {
-    body = <div className="panel-resizer--empty">{hiddenText}</div>
+    if (resizingEnabled && visibility === 'hidden') {
+      body = <div className="panel-resizer--empty">{hiddenText}</div>
+    }
   }
 
+  const klass = classnames('panel-resizer', {
```

Comment:
Lolwut

Reply:
omg idk it's an old habit for getting around reserved keywords. what's our convention?

Reply:
I just give it a longer name, like `resizerClass`